### ISSUE TITLE
Is it real to control venicle on Windows by API? 

### ISSUE THREAD

**Issue by eniseiskaya** [lgsvl/simulator#794]
Hello
 Is it real to control venicle on Windows by API? I need to launch from unity and control the vehicle.

**Comment by ntutangyun** [comment]
yes it's possible. Please check out the docs on [Python API](https://www.lgsvlsimulator.com/docs/python-api/)

**Comment by eniseiskaya** [comment]
@ntutangyun For some reason, the example from the documentation did not work. I redid it and now it seems to be running. Thank you.
 Have you ever had such a bug? Is it possible to move the vehicle higher?
![Скриншот 07-06-2020 18_37_48](https://user-images.githubusercontent.com/55806370/83974200-38674500-a8f4-11ea-975c-011ca700114c.png)

**Comment by ntutangyun** [comment]
@eniseiskaya you could try setting the `z` in `AgentState` to a higher value when spawning the ego vehicle.

**Comment by EricBoiseLGSVL** [comment]
@eniseiskaya Please see #778
